LICENSE.md

```diff
@@ -11,7 +11,7 @@
 7. Semantic frames (see `@frame`).
 8. Participant referents (see `@subjref`, `@participantref`)
 9. Synonyms (see `./sources/Clear/synonyms`)
-10. Word Senses (see `@SenseNumber`)
+10. Word Senses (see `@SenseNumber`) that are distinct from the Semantic Dictionary of Biblical Hebrew word senses.
```

Comment:
This is fine, but adding a phrase that positively defines what these are might be helpful. Example: 

> statistically generated, distinct from those found in the Semantic Dictionary of Biblical Hebrew

Reply:
I had the same thought re positive identification.